src/settings_import_export.py

```diff
@@ -39,19 +40,29 @@ def import_settings(path):
                     subprocess.run(['gsettings', 'set', \
                         'io.github.fsobolev.Cavalier', line.split(' ')[0], \
                         line.replace(line.split(' ')[0], '').strip()])
+        toast_msg = 'Settings sucessfully imported'
```

Comment:
Please use gettext for all toast messages strings, put them inside `_()` (e.g. `_('Settings sucessfully imported)`)